responses/14_troubleshooting.md

```diff
@@ -0,0 +1,32 @@
+# Troubleshooting: Importing a course
+
+If you're having trouble importing your course, here are a few tips for troubleshooting.
+
+## I can't find the place to import
+
+There are a few different ways to start a course:
+
+- Through the Learning Lab course builder
+- Manually
+
+If you do this manually, follow the instructions in the ["Create the course on Learning Lab (manual option"](https://lab.github.com/docs/course-ownership-and-repositories#manual-repository-setup) paragraph.
+
+This should lead you to a URL that looks like `https://lab.github.com/:owner/new`. _(Ex: https://lab.github.com/githubtraining/new)_
+
+## My import fails and I don't know why
+
+### Validate the YAML
+
+Try to validate the YAML in the `config.yml` using a tool like [yamllint](http://www.yamllint.com/).
+
+### Check response files
```

Comment:
@brianamarie I wonder if there are a few more common issues we could add here, like the one I ran into with not having actions for each step? @JasonEtco are the importer errors stored anywhere? Maybe we could use that to see what's most common.

Reply:
I'll add in your example in 4bf3228 @crichID, and we can add others based on @JasonEtco's feedback.